schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -41,8 +41,9 @@
               {
                 "oneOf": [
                   { "$ref": "http://schema.management.azure.com/schemas/2016-06-01/Microsoft.RecoveryServices.json#/resourceDefinitions/vaults" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-07-01-preview/Microsoft.ServerManagement.json#/resourceDefinitions/node" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-07-01-preview/Microsoft.ServerManagement.json#/resourceDefinitions/gateway" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-07-01-preview/Microsoft.ServerManagement.json#/resourceDefinitions/nodes" },
```

Comment:
I did not find /resourceDefinitions/nodes, /resourceDefinitions/gateways, /resourceDefinitions/sessions in the Microsoft.ServerManagement.json file,
have you used the validateJSON.js and runSchemaTests.js to check the schema and tests? Those shall catch the issue like this.